pydantic_extra_types/country.py

```diff
@@ -18,1762 +25,21 @@ class CountryInfo:
     alpha3: str
     numeric_code: str
     short_name: str
-    official_name: str
+    # NOTICE: Not all countries have an official name
+    official_name: Optional[str]
```

Comment:
It doesn't look like it can assume `None` tho? I see empty string, but not None.